packages/app-desktop/gui/NewVersionDialog/style.scss

```
@@ -0,0 +1,46 @@
+			margin-top: 8px;
+
+			.changelog-label {
+				font-weight: bold;
+				margin-bottom: 8px;
+			}
+
+			.changelog-text {
+				max-height: 280px;
+				overflow-y: auto;
+				white-space: pre-wrap;
+				word-wrap: break-word;
```

Stylelint (17.3.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)